Packs/Jira/Integrations/JiraV3/JiraV3.py

```diff
@@ -1123,6 +1143,25 @@ def get_all_projects(self) -> List[Dict[str, Any]]:
             url_suffix='rest/api/2/project'
         )
 
+    def issue_get_forms(self, issue_id: str) -> List:
+        """Makes an API call to pull down all forms their data for an issue
+
+        :param issue_id: Issue to pull forms for
+        :type issue_id: str
+        :return: The raw response and a cleaned up response
+        :rtype: tuple[List, List]
+        """
+        j_res = self.http_request(
+            method='GET',
+            url_suffix=f'rest/proforma/api/2/issues/{issue_id}/forms',
```

Comment:
`proforma`? 

Lets stay consistent with the endpoints used in the rest of the commands, for reference see: 
https://github.com/demisto/content/blob/abbd5d40ae4e4837ba454bea853b1c01ab67c388/Packs/Jira/Integrations/JiraV3/JiraV3.py#L243

https://github.com/demisto/content/blob/abbd5d40ae4e4837ba454bea853b1c01ab67c388/Packs/Jira/Integrations/JiraV3/JiraV3.py#L525

Reply:
The proforma API endpoints are part of a common plugin for Jira. The version isn't going to match the server, the version is 2 for this plugin. I could add a `self.proforma_api_version` which would then match the setup of the other endpoints.

Reply:
Fixed